lib/grunt/task.js

```diff
@@ -1,5 +1,11 @@
 'use strict';
 
+// Keep track of the number of log.error() calls.
+var errorcount,
+
+  // The last specified tasks message.
+  lastInfo;
```

Comment:
Ah cool. Thanks for pointing that out. Could we put the declarations onto one line and just concat the code comments with `and`?